Sources/WebDriver/Requests.swift

```diff
@@ -540,20 +540,68 @@ public enum Requests {
             }
         }
     }
+  
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidtouchdoubleclick
+    public struct SessionTouchDoubleClick: Request {
+        public var session: String
+        public var element: String 
+
+        public var pathComponents: [String] { ["session", session, "touch", "doubleclick"] }
+        public var method: HTTPMethod { .post }
+        public var body: Body { .init(element: element) }
+
+        public struct Body: Codable {
+            public var element: String
+        }
+    }
+
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#sessionsessionidtouchflick
+    public struct SessionTouchFlickElement: Request {
+        public var session: String 
+        public var element: String
+        public var xOffset: Double
+        public var yOffset: Double
+        public var speed: Double
+
+        public var pathComponents: [String] { ["session", session, "touch", "flick"] }
+        public var method: HTTPMethod { .post }
+        public var body: Body { .init(xOffset: xOffset, yOffset: yOffset, speed: speed) }
+
+        public struct Body: Codable {
+            public var xOffset: Double
```

Comment:
According to spec, these properties should serialize as lower case `xoffset`/`yoffset`. We'll need custom CodingKeys to achieve this.